ui/components/ui/form-combo-field/form-combo-field.tsx

```diff
@@ -176,7 +177,7 @@ export default function FormComboField({
       setDropdownVisible(false);
 
       if (option) {
-        handleChange({ target: { value: option.primaryLabel } });
+        handleChange({ target: { value: option.value } });
```

Comment:
Since `value` is optional, should we fallback to the `primaryLabel` which is always available?

Reply:
I've set `value` to required.
It's a good idea to separate the display text from the values we use programmatically, similar to the approach with HTML <option> tags. We can easily update the UI without affecting the underlying logic and vice versa. It also helps with internationalization efforts.